library/Registry.md

```diff
@@ -0,0 +1,22 @@
+# Q# Package Registry
+
+This document is a list of all known (to us) Q# packages. If you have a package that you would like to add to this list, please open a pull request.
```

Comment:
I would err on the side of saying "notable" rather than "all known (to us)" in there. Also, did we want to use the work "package" or "project" here? I think we were trying to get away from the implications of "package." Maybe "library projects" is better? Tried it with both changes below.
```suggestion
This document is a list of notable Q# library projects. If you have a package that you would like to add to this list, please open a pull request.
```